BGAnimations/ScreenSelectMusic overlay/PerPlayer/over.lua

```diff
@@ -1,10 +1,20 @@
 local t = Def.ActorFrame{}
 
--- always add StepArtist and PaneDisplay actors for both players, even if only one is joined right now
+-- always add DensityGraph, StepArtist, and Cursor actors for both players, even if only one is joined right now
 -- if the other player suddenly latejoins, we can't dynamically add more actors to the screen
 -- we can only unhide hidden actors that were there all along
 for player in ivalues( PlayerNumber ) do
-	-- bouncing cursor inside the grid of difficulty blocks
+	-- Currently the Density Graph doesn't work for Courses.
+	-- Disable the functionality.
+	if not GAMESTATE:IsCourseMode() then
+		t[#t+1] = LoadActor("./DensityGraph.lua", player)
+	end
+	-- AuthorCredit, Description, and ChartName associated with the current stepchart
+	t[#t+1] = LoadActor("./StepArtist.lua", player)
```

Comment:
Is there any difference between this being in over or under?

Reply:
I don't think so since there aren't any overlapping elements. I originally put it in over since I was keeping the original elements intact below it as I was working. We could probably just combine the under and over files if we want to. WDYT?

Reply:
I'd probably put all of it in under.

Reply:
I removed over.lua, moved everything to under.lua, and then renamed under.lua to default.lua